interview-questions.md

```diff
@@ -7,81 +7,83 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. Why would you use the method `super()`?
 
-  Your answer:
+  Your answer: Super is placed in the constructor of a child's React component so that it can have access to its parent's props.
```

Comment:
Yes! But something to consider is that super is not just used for React, its used in Classes in general!